cmd/unlocker/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"crypto/tls"
+	"flag"
+	"fmt"
+	"github.com/howeyc/gopass"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+)
+
+var (
+	Version    = "No version provided"
+	certFile   = flag.String("cert", "client.pem", "A PEM eoncoded certificate file.")
+	keyFile    = flag.String("key", "key.pem", "A PEM encoded private key file.")
+	targetHost = flag.String("targetHostPort", "www.example.com", "The hostname/port for .")
```

Comment:
How about two parameters: keymasterHostname (string with no default) and keymasterPortNum (uint, with default)?